flake.nix

```diff
@@ -163,6 +163,7 @@
       f here rhs lhs;
   in
     with yants "data-merge"; {
+      __functor = self: arg: self.merge arg;
```

Comment:
`__functor = self: self.merge;` is a bit more concise, no?

Reply:
yeah beta reduction, duh :facepalm: